Aquarius/Aquarius.cpp

```diff
@@ -0,0 +1,36 @@
+#include "Aquarius.h"
+
+INITIALIZE_EASYLOGGINGPP
+
+int Aquarius::Test::testMain()
```

Comment:
Just curious, when implementing classes should we just do: 

```
namespace aquarius {
Test::Main() 
{
}

}
```

Right?

Reply:
Correct